platform/viewer/src/lib/DICOMSR/srUtils.js

```diff
@@ -0,0 +1,71 @@
+import OHIF from '@ohif/core';
+
+const { studyMetadataManager } = OHIF.utils;
+
+const supportedSopClassUIDs = [
+  '1.2.840.10008.5.1.4.1.1.88.22',
+  '1.2.840.10008.5.1.4.1.1.11.1',
+];
+
+const getAllDisplaySets = () => {
+  const allStudies = studyMetadataManager.all();
+  let allDisplaySets = [];
+
+  allStudies.forEach(study => {
+    if (study.getDisplaySets) {
+      allDisplaySets = allDisplaySets.concat(study.getDisplaySets());
+    }
+  });
+
+  return allDisplaySets;
+};
```

Comment:
This isn't really an SR util. Should this be a part of the studyMetadataManager?

Reply:
If not, can we pass in the `studyMetadataManager` instead of importing?

Reply:
Done 👍